.github/workflows/steward.yml

```diff
@@ -16,15 +16,15 @@ jobs:
         id: generate-token
         uses: tibdex/github-app-token@v1
         with:
-          app_id: 123456
+          app_id: ${{ secrets.STEWARD_APP_ID }}
           private_key: ${{ secrets.STEWARD_PRIVATE_KEY }}
 
       - name: Launch Scala Steward
         uses: scala-steward-org/scala-steward-action@v2
         with:
           github-token: ${{ steps.generate-token.outputs.token }}
           branches: series/0.23,main
-          author-email: ${{ secrets.STEWARD_APP_ID }}+typelevel-steward[bot]@users.noreply.github.com
+          author-email: 106827141+typelevel-steward[bot]@users.noreply.github.com
           author-name: typelevel-steward[bot]
           github-app-id: ${{ secrets.STEWARD_APP_ID }}
           github-app-key: ${{ secrets.STEWARD_PRIVATE_KEY }}
```

Comment:
This is the configuration that enables steward to automatically get a list of repositories to update. If we remove it, then we have to use the `repos.md`. This has the advantage that it supports branches and is adversary-proof, since we just won't add them to the list. Any repo that wants updates needs to install the app and add itself to the list.